hack/verify-api-descriptions.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ "$#" -lt 1 ]; then
+	echo "Usage: verify-api-descriptions.sh <OpenShift API version>"
+    exit
+fi
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
```

Comment:
Do we do this elsewhere?  I'm not a huge fan of changing directories in the shell process.